src/main/scala/io/scalac/tezos/translator/model/Translation.scala

```diff
@@ -0,0 +1,30 @@
+package io.scalac.tezos.translator.model
+
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+
+object Translation {
+
+  sealed trait From
```

Comment:
it's ok for me

 but FYI there is nice library for such cases: https://github.com/lloydmeta/enumeratum#usage